src/libraries/System.Reflection.Emit/tests/ModuleBuilder/ModuleBuilderDefineType.cs

```diff
@@ -10,7 +10,7 @@ public class ModuleBuilderDefineType
     {
         public static IEnumerable<object[]> TestData()
         {
-            foreach (string name in new string[] { "TestName", "testname", "class", "\uD800\uDC00", "a\0b\0c" })
```

Comment:
Turns out it was failing on mono only for `"a\0b\0c"` case and @GrabYourPitchforks suggested to remove it as it is not valid scenario to test